### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by gidsi**
i've created the mail alias temporarily, it's a forward to the mailboxes of @dbrgn @rnestler @slopjong and @gidsi

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at spaceapi-team@chaospott.de. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```
Comment (rnestler) [inline CODE_OF_CONDUCT.md:37]: Should we mention chat.spaceapi.net here as well as a way to contact the project team?

Comment (dbrgn) [reply CODE_OF_CONDUCT.md:37]: For this type of contact I'd stick to e-mail.

**Review by dbrgn — APPROVED** [review]
Good to go from me. I think this needs an explicit approval from every team member before we merge it though.

**Comment by dbrgn** [comment]
By the way, merging this into the codebase is a bit moot if the domain points to the other server now :)

**Review by rnestler — APPROVED** [review]
:+1:

**Comment by rnestler** [comment]
Should we merge this here? Or create a separate repository or create a spaceapi.io repository and add it there?

**Comment by dbrgn** [comment]
I'd add it to a new `website` repo. I'll create it in a few minutes with the source from the spacedirectory website.

**Comment by dbrgn** [comment]
@rnestler @gidsi one of you can now move it to https://github.com/SpaceApi/website/!

**Pull request closed without merging**